components/css/custom.css

```diff
@@ -92,3 +92,109 @@ span.col-14, span.col-15 {
   text-decoration: none;
 }
 
+.select-by-menu {
+  padding-bottom: 10px;
+}
+
+.react-calendar {
+  width: 350px;
+  max-width: 100%;
+  background: white;
+  border: 1px solid #a0a096;
```

Comment:
All of the react-calendar CSS is boilerplate code that came with that package. I'm not experienced with Sass or styled-components, but I will read up on them and I'm interested in using them for this or future projects! Happy to talk about it some more.

Reply:
ah, I gotcha, that makes sense. In that case, we should make it clear that those styles are straight from the vendor and not our own "custom" css to modify it. Can we put whatever boilerplate we didn't write in a file like `components/css/vendor/react-calendar.css` and import it into the component where we need it?

Reply:
I initially tried that but was having problems because when I try to import the new CSS file using a command like `import "../../../components/css/vendor/react-calendar.css";` I get the error "Cannot find module ... at webpackMissingModule. The only solution I could find was dumping the CSS into our custom.css file. It seems like I'm either not importing the file right of webpack by default looks for CSS in that custom.css file. If it's the latter, we may need to do something to prevent it from taking that action. Do you know of a better way to fix that? If it's not worth the time, we could add a comment in the CSS file saying everything below is from react-calendar, but that's not very elegant. Thoughts?

Reply:
maybe changing something webpack.config.js would fix the issue?